app/assets/styles/base/breadcrumbs.scss

```diff
@@ -60,14 +60,22 @@ bex-breadcrumbs {
             word-break: break-all;
             white-space: nowrap;
             color: $whiteSmoke;
+            font-size: 12px;
+            line-height: 16px;
+
+            &.no-label {
+                height: 100%;
+                display: flex;
+                align-items: center;
+            }
         }
 
         > div.crumb-label {
             color: $whiteSmoke-darker;
             font-style: italic;
             text-align: right;
             line-height: 0.7em;
-            font-size: 0.7em;
+            font-size: 9px;
```

Comment:
made the text smaller and I don't like actually using the non fixed number for font because sometimes you get blurry text